appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/palette/YoungAndroidPalettePanel.java

```diff
@@ -192,6 +200,25 @@ public void onChange(ChangeEvent event) {
       }
     });
 
+    /* User presses the slash key, the search text box is focused */
+    RootPanel.get().addDomHandler(new KeyDownHandler() {
```

Comment:
YoungAndroidPalettePanel is not currently a singleton, so this will end up binding a new handler for every open screen. Make sure to test with multiple screens to make sure the behavior is correct.

Reply:
This does work as expected.